src/webgpu/api/validation/createComputePipeline.spec.ts

```diff
@@ -0,0 +1,164 @@
+export const description = `
+createComputePipeline and createComputePipelineAsync validation tests.
+
+TODO:
+For createComputePipeline and its async version createComputePipelineAsync,
```

Comment:
We usually use these TODOs to say what work should be done in the file. Everything that's in this TODO is implemented so it doesn't need to appear. Is there something else that should be tested?

Reply:
I sorry I meant that we could remove the test plan as well, since it is now in the `.desc()` for each test.